tools/README.md

```diff
@@ -167,3 +167,70 @@ The output chunked graph metadata will go as follows (assuming the current direc
     "edge_data": {}
 }
 ```
+
+## Change edge type to canonical edge type for partition configuration json
+
+`change_etype_to_canonical_etype.py` is a tool that helps you convert etypes to canonical_etypes in a partition configuration file, and it overwrites the original file on disk. 
```

Comment:
**Note:** In the upcoming DGL v1.0, we will require the partition configuration file to contain only canonical edge type. This tool is designed to help migrating existing data partitions from old style to new one.